syntax = "proto3";
import "app/proto/messages.proto";
service TeamRecommender{
  rpc AskForTeam(User) returns (TeamRecommendation);
}
